test/e2e/common/configmap_volume.go

```diff
@@ -720,8 +722,14 @@ func doConfigMapE2EWithMappings(f *framework.Framework, uid, fsGroup int64, item
 		},
 	}
 
-	if userID != 0 {
-		pod.Spec.SecurityContext.RunAsUser = &userID
+	if asUser {
```

Comment:
nit: factor out this block to a function since it's copied 4 times in this PR.

Reply:
Done